lib/tasks/projects.rake

```diff
@@ -73,15 +73,12 @@ namespace :projects do
     puts "Mediaflux asset #{asset_id} updated"
   end
 
-  desc "Outputs to a file an aterm script to create a project (namespace and collection) in Mediaflux"
+  desc "Outputs to the console the Aterm script to create a project in Mediaflux"
   task :create_script, [:project_id] => [:environment] do |_, args|
     project_id = args[:project_id]
-    service = MediafluxScriptFactory.new(project_id: project_id)
-    script = service.build_create_script
-
-    file_name = "project_create_#{project.id}.txt"
-    puts "Saving script to #{file_name}"
-    File.write(file_name, script)
+    project = Project.find(project_id)
+    service = MediafluxScriptFactory.new(project: project)
+    puts service.aterm_script
```

Comment:
I might rather this still create a file by default.  Is there a reason for this change?